_includes/menu-item-ids/table-menu-items.md

```diff
@@ -11,13 +11,17 @@
 | `column`                  | Table                                                                                                    | Column menu item with related controls.                                   |
 | `tableinsertcolumnbefore` | Table                                                                                                    | Insert column before the currently selected column {{site.requires_5_4v}} |
 | `tableinsertcolumnafter`  | Table                                                                                                    | Insert column after the currently selected column {{site.requires_5_4v}}  |
-| `tabledeletecolumn`       | Table                                                                                                    | Deletes curently selected column or columns {{site.requires_5_4v}}        |
+| `tablecutcolumn`          | Table                                                                                                    | Cut the currently selected column or columns {{site.requires_5_5v}}           |
```

Comment:
This isn't technically correct. They are available from 5.4 onwards, we just didn't include the enterprise translations from 5.5. Happy to leave this as is, just commenting in case you want to change it.

Reply:
we'll say 5.5 for sanity's sake